packages/@aws-cdk/aws-docdb/lib/database-secret.ts

```diff
@@ -68,8 +77,10 @@ export class DatabaseSecret extends Secret {
           masterarn: props.masterSecret?.secretArn,
         }),
         generateStringKey: 'password',
-        excludeCharacters: props.excludeCharacters ?? '"@/',
+        excludeCharacters: excludedCharacters,
       },
     });
+
+    this.excludedCharacters = excludedCharacters;
```

Comment:
We can simplify this code:

```ts
  constructor(scope: Construct, id: string, props: DatabaseSecretProps) {
    this.excludedCharacters = props.excludeCharacters ?? '"@/';

    super(scope, id, {
      // ...
        excludeCharacters: this.excludedCharacters,
      },
    });
```

Reply:
no we can not ;-) at least I cannot access this before the super call?
But I need this variable inside the super call ... so kind of chicken or the egg problem?

That was the reason I firs assigned the default to a const and later made it accessible in the scope of the class.